data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/request/command/DeleteApprovalRule.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "record",
+  "name": "DeleteApprovalRule",
+  "namespace": "net.corda.data.membership.db.request.command",
+  "doc": "Delete a previously persisted approval rule",
+  "fields": [
+    {
+      "name": "ruleId",
+      "doc": "The ID of the rule to be deleted",
```

Comment:
This is just a nit: but can we put a dot at the end of each doc lines? We did that before in our schemes and it would be nice to follow the same pattern.